### ISSUE TITLE
 Error loading shared library ld-linux-x86-64.so.2: No such file or directory

### ISSUE THREAD

**Issue by Rivers-King** [jhgan00/java-ort-example-yolov5#11]
Hello, I'd like to ask you another question. When I try to deploy the project to a CentOS system, it reports an error as shown in the image below. However, I have the corresponding files for my dependencies. Do you know how to resolve this issue?

![image](https://github.com/jhgan00/java-ort-example-yolov5/assets/140472263/fdda99b5-e549-499a-bd0b-221149d464fe)

![1691047916396](https://github.com/jhgan00/java-ort-example-yolov5/assets/140472263/b0cfa827-d1b9-4806-bdc3-a22f7c6ad2ac)

my system version is CentOS 7.9.2009 x86_64(Py3.7.9)

**Comment by jhgan00** [comment]
Hello. I also cannot pinpoint the exact cause accurately. To reproduce the issue, I attempted to compile and run on a CentOS7 image, but I did not encounter the same error you provided; instead, the execution failed due to a different issue (GLIBC version problem). I apologize for not being able to assist.

**Comment by Rivers-King** [comment]
Thank you for your response. From what I understand, the issue I encountered was also caused by a GLIBC version problem. However, due to the presence of other projects on the deployed system, and to prevent any potential impact of a GLIBC version upgrade on those projects, I made the decision to switch servers. This transition was successful.

The new server version is Ubuntu 20.04.6 LTS x86_64 (Python 3.7.9), and the problem is no longer present. Once again, I sincerely appreciate your assistance.
